zakra/scss/layout/_footer.scss

```diff
@@ -0,0 +1,62 @@
+.footer {
+    background-color: $secondaryColor;
+    padding: 50px 0;
+}
+
+.footer p {
+    color: $primaryColor;
+}
+
+.contents-list {
+    color: $primaryColor;
+    width: 25%;
+    font-size: 18px;
+}
+
+.contents-list .flex-box {
+    justify-content: left;
+}
+
+.contents-list li {
+    padding: 10px;
+}
+
+.footer-contents li {
+    list-style-type: none ;
+}
+
+.footer-contents {
+    padding: 0;
+}
+
+.about-sec {
+    line-height: 1.5;
+    padding: 10px;
+}
+
+.footer-sec-one {
+    padding: 30px 0;
+    border-bottom: 1px solid $textColor;
+}
+
+/*****************MEDIA QUERIES******************/
+
+@media (max-width: 1000px) {
+    .footer-sec-one .flex-box {
+        flex-direction: column;
+    }
+
+    .contents-list .flex-box {
+        flex-direction: row;
+    }
+
+    .contents-list ul {
+        padding: 0;
+    }
+
+    .contents-list {
+        margin-bottom: 20px;
+        width: 100%;
+    }
+}
+
```

Comment:
**Create a mixin for media queries and include it within specific styles.**

// Breakpoint viewport sizes and media queries.
// A map of breakpoints.
$breakpoints: ( xs: 360px, md: 769px, lg: 992px, xl: 1401px);
//  RESPOND ABOVE
// @include respond-above(sm) {}
@mixin respond-above($breakpoint) {
    // If the breakpoint exists in the map.
    @if map-has-key($breakpoints, $breakpoint) {
        // Get the breakpoint value.
        $breakpoint-value: map-get($breakpoints, $breakpoint);
        // Write the media query.
        @media (min-width: $breakpoint-value) {
            @content;
        }
        // If the breakpoint doesn't exist in the map.
    }
    @else {
        // Log a warning.
        @warn 'Invalid breakpoint: #{$breakpoint}.';
    }
}

//  RESPOND BELOW
// @include respond-below(sm) {}
@mixin respond-below($breakpoint) {
    // If the breakpoint exists in the map.
    @if map-has-key($breakpoints, $breakpoint) {
        // Get the breakpoint value.
        $breakpoint-value: map-get($breakpoints, $breakpoint);
        // Write the media query.
        @media (max-width: ($breakpoint-value - 1)) {
            @content;
        }
        // If the breakpoint doesn't exist in the map.
    }
    @else {
        // Log a warning.
        @warn 'Invalid breakpoint: #{$breakpoint}.';
    }
}

//  RESPOND BETWEEN
// @include respond-between(sm, md) {}
@mixin respond-between($lower, $upper) {
    // If both the lower and upper breakpoints exist in the map.
    @if map-has-key($breakpoints, $lower) and map-has-key($breakpoints, $upper) {
        // Get the lower and upper breakpoints.
        $lower-breakpoint: map-get($breakpoints, $lower);
        $upper-breakpoint: map-get($breakpoints, $upper);
        // Write the media query.
        @media (min-width: $lower-breakpoint) and (max-width: ($upper-breakpoint - 1)) {
            @content;
        }
        // If one or both of the breakpoints don't exist.
    }
    @else {
        // If lower breakpoint is invalid.
        @if (map-has-key($breakpoints, $lower)==false) {
            // Log a warning.
            @warn 'Your lower breakpoint was invalid: #{$lower}.';
        }
        // If upper breakpoint is invalid.
        @if (map-has-key($breakpoints, $upper)==false) {
            // Log a warning.
            @warn 'Your upper breakpoint was invalid: #{$upper}.';
        }
    }
}